### PR TITLE
use navigate and route, clean up code

### PR DESCRIPTION
**Pull request by brucewzj99**
# Description

Please include a summary of the changes and the related issue. 
Please also include relevant screenshot and context. 

Fixes # (issue)
Clean up code, using of routerlink, navigate
Fix account creation

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Register and login of account

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: SmartHomeManager/SmartHomeManager.View/src/pages/account/Login.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { React, useState } from 'react';
+import { useNavigate } from "react-router-dom";
+import { Link as RouterLink } from "react-router-dom";
```
Comment (fabianchua6) [inline SmartHomeManager/SmartHomeManager.View/src/pages/account/Login.js:3]: can combine into one line

File: SmartHomeManager/SmartHomeManager.View/src/pages/account/Register.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import { React, useState } from 'react';
+import { useNavigate } from "react-router-dom";
+import { Link as RouterLink } from "react-router-dom";
```
Comment (fabianchua6) [inline SmartHomeManager/SmartHomeManager.View/src/pages/account/Register.js:3]: here as well

**Review by fabianchua6 — CHANGES_REQUESTED** [review]
noiceee

**Comment by brucewzj99** [comment]
Need to make more chances

**Review by DoubleBounce — APPROVED** [review]
LGTM

**Review by fabianchua6 — APPROVED** [review]

**Pull request merged**